nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -1240,6 +1258,18 @@ private void addGuaranteedNonNullFromInvokes(
       if (privMethodElem != null) {
         result.add(privMethodElem);
       }
+      // Hack: If we see a try{...}finally{...} statement, without a catch, we consider the methods
+      // inside both blocks
+      // as "top level" for the purposes of finding initialization methods. Any exception happening
+      // there is also an
+      // exception of the full method.
+      if (stmt.getKind().equals(Tree.Kind.TRY)) {
+        TryTree tryTree = (TryTree) stmt;
+        if (tryTree.getCatches().size() == 0) {
+          result.addAll(getSafeInitMethods(tryTree.getBlock(), classSymbol, state));
+          result.addAll(getSafeInitMethods(tryTree.getFinallyBlock(), classSymbol, state));
```

Comment:
Hi, I noticed this and I encountered a NullPointerException here. I have a try-with-resources block without a finally block and it causes a NPE on line 1255 because `blockTree` is `null`. I filed an issue in regards to this: https://github.com/uber/NullAway/issues/133